src/library.js

```diff
@@ -877,6 +877,9 @@ LibraryManager.library = {
       ENV['PWD'] = '/';
       ENV['HOME'] = '/home/web_user';
       ENV['LANG'] = 'C.UTF-8';
+      if (navigator.languages != 'undefined' && navigator.languages.length > 0) {
+          ENV['LANG'] = navigator.languages[0].replace('-', '_') + '.UTF-8';
+      }
```

Comment:
This change seems a little risky to me - maybe some language has `-` in other places? Also, the app may behave differently for different users, which is the point I guess :) but it may be surprising to start doing so now.

Reply:
To improve Closure minifiability, one could do:
```js
ENV['LANG'] = (typeof navigator === 'object' && navigator.languages && navigator.languages[0] || 'C').replace('-', '_') + '.UTF-8';
```
(I would assume that if `navigator.languages` exists, it is an array)